pkgs/jnigen/test/simple_package_test/generate.dart

```diff
@@ -63,15 +63,15 @@ void compileJavaSources(String workingDir, List<String> files) async {
   }
 }
 
-Config getConfig() {
+Config getConfig([SummarizerBackend backend = SummarizerBackend.asm]) {
```

Comment:
Prefer named arguments over positional (optional positional arguments always end up being a mistake in my experience).